graph/community/louvain_directed.go

```diff
@@ -353,25 +352,25 @@ func (g *ReducedDirected) To(v graph.Node) []graph.Node {
 func (g *ReducedDirected) HasEdgeBetween(x, y graph.Node) bool {
 	xid := x.ID()
 	yid := y.ID()
-	if xid == yid {
+	if xid == yid || xid != int64(int(xid)) || yid != int64(int(yid)) {
```

Comment:
It seems to be rather involved to keep the community node as an `int` rather than an `int64`. May I ask, what is the main rationale for keeping it `int` than updating it to be consistent with node ID as `int64`, as is done elsewhere. This change will inevitably introduce API breakage. Might as well do one swooping change to `int64`, and keep the code simple and consistent?